docs/.vuepress/styles/index.scss

```diff
@@ -187,13 +200,67 @@
         }
     }
     
+    // Table of Contents
     #toc {
-        padding: 0 9rem !important;
-    }
-    
-    @media screen and (max-width: 1600px) {
-        #toc {
-            padding: 0 4rem !important;
+        padding: 0 0 0 2rem;
+        max-width: calc(50vw - 50% - 6rem); // Fill right-side of content
```

Comment:
this feels brittle.  Don't fully understand what we're tyring to accomplish on this?

Reply:
This is because the alternative is to provide static em measurements of width based off max screen resolutions (which caused the problem in the first place)

This isn't too brittle considering it's an absolutely positioned section injected in by VuePress (little control there)

Reply:
@karnie6 This Before/After GIF better shows what I've described above

https://github.com/matter-labs/zksync-web-era-docs/assets/1890113/b05eb09f-a2c3-4b85-b3c1-b70ea7214b0c


https://github.com/matter-labs/zksync-web-era-docs/assets/1890113/cabad6d8-15e1-4220-b537-3fa1d03cd3da